Jenkinsfile

```diff
@@ -19,7 +19,7 @@ pipeline {
         }
         stage("Integration Tests") {
             steps {
-                sh "./gradlew integrationTest"
+                sh "./gradlew solrIntegrationTest"
```

Comment:
Old `integrationTest` tests are ignored now?

Reply:
They are included in solrIntegrationTest

Reply:
as in: it should be the other way around: `solrIntegrationTest` should be included by `integrationTest`